forms-flow-admin/src/variables.scss

```diff
@@ -0,0 +1,4 @@
+$dark: var(--fwf-dark);
```

Comment:
Can we use ff as the key, short for **f**orms**f**low

Reply:
Yes, we can.